website/source/docs/connect/connect-internals.html.md

```diff
@@ -46,7 +46,7 @@ also ships with built-in support for [Vault](/docs/connect/ca/vault.html). The P
 and can be extended to support any system by adding additional CA providers.
 
 All APIs required for Connect typically respond in microseconds and impose
-minimal overhead to existing services. This is because the Connect-related APIs
+minimal overhead to existing services. Ensuring this, Connect-related APIs calls
```

Comment:
I see why the old sentence is a bit awkward her but perhaps we could simplify the grammar/voicing further? 

```suggestion
minimal overhead to existing services. To ensure this, Connect-related API calls
```

Reply:
Agreed!